src/modules/EnsonoBuild/exported/Build-DockerImage.ps1

```diff
@@ -144,14 +162,19 @@ function Build-DockerImage() {
     Write-Information -MessageData ("No tag has been specified for the image, a default one has been set: {0}" -f $tag)
   }
 
+  # set a default for the platforms if none have been set
+  if ([string]::IsNullOrEmpty($platforms)) {
+    $platforms = @("linux/arm64", "linux/amd64")
```

Comment:
This probably shouldn't be in this command as a default? Most clients will only want to build apps for one architecture by default? (the change should be made as a pass in from `stacks-docker-images`? This affects _all_ images built going forwards for _all_ people who use this?

Reply:
Good point
Have updated to detect the platform the user is on and will build accordingly